source/jormungandr/tests/routing_tests.py

```diff
@@ -407,6 +407,16 @@ def test_novalidjourney_on_first_call_debug(self):
         eq_(response['journeys'][1]['arrival_date_time'], "20120614T160000")
         eq_(response['journeys'][1]['type'], "best")
 
+    def test_codes_on_object(self):
```

Comment:
can you add a comment on the method ?

And maybe add a method that do the same test but with `show_codes=false`